hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusGetTopicInfoHandler.java

```diff
@@ -165,7 +169,7 @@ private Optional<ConsensusTopicInfo> infoForTopic(
                     .ifPresent(account ->
                             info.autoRenewAccount(AccountID.newBuilder().accountNum(account)));
 
-            info.ledgerId(queryContext.getLedgerId());
+            info.ledgerId(networkInfo.ledgerId());
```

Comment:
networkInfo can be null

Reply:
It should not. A `networkInfo` always has to be provided during construction. The empty constructor is just left in there for now to make the project compile.